src/app/qgisapp.cpp

```diff
@@ -3692,10 +3691,10 @@ void QgisApp::restoreWindowState()
 ///////////// END OF GUI SETUP ROUTINES ///////////////
 void QgisApp::sponsors()
 {
-  QgsSponsors *sponsors = new QgsSponsors( this );
-  sponsors->show();
-  sponsors->raise();
-  sponsors->activateWindow();
+  QgsSettings settings;
+  QString qgisSponsorsUrl = settings.value( QStringLiteral( "qgis/qgisSponsorsUrl" ),
+                            tr( "http://qgis.org/en/site/about/sponsorship.html" ) ).toString();
```

Comment:
I think yes, for example for closed networks. Also this done for all other URLs, e.g. bugtracker, homepage